docs/README.md

```diff
@@ -29,9 +29,30 @@ The VTEX LocaleSwitcher is a VTEX component capable of changing the current lang
 },
 ```
 
+### locale-swicher-list
+
+This block doesn't have any prop, you just have to use it and pass the `locale-switcher-item` as `children` of it.
+
+### locale-switcher-item
+
+The options of languages. This block accepts a `label` or you can pass `children` to it in case you want to render something like a country flag.
+
+| Prop name | Type | Description | Default value |
+| --- | --- | --- | --- |
+| `label` | `string` | Label of the item e.g. `EN, PT, JA, ES`... | `undefined` |
+| `localeId` | `string` | The id of the locale | - |
+| `display` | `DisplayMode` | The id of the locale | `'default'` |
```

Comment:
```suggestion
| Prop name | Type | Description | Default value |
| --- | --- | --- | --- |
| `label` | `String` | Language label e.g. `EN`, `PT`, `JA`, `ES`. | `undefined` |
| `localeId` | `String` | Locale ID e.g. `en-US`, `pt-BR`, `ja-JP`, `es-ES`.  | `undefined` |
| `display` | `Enum` | Whether the Locale ID element will be rendered (`default`) or not (`none`). | `default` |
```

Reply:
I didn't understand the `display` prop functionality. What is the Locale ID element? I think this should be more clarified here.